yarn-project/noir-contracts/src/contracts/child_contract/src/main.nr

```diff
@@ -25,14 +28,16 @@ contract Child {
         context.finish(inputs)
     }
 
-    // Nested public functions always get called with MAX_ARGS, since we don't have the ABI available
+    // Returns base_value + 42.
+    // @param _padding: Nested public functions always get called with MAX_ARGS, since we don't have the ABI available
     // during execution time to know how many args are expected, hence the _padding argument. We should
```

Comment:
nit: you can remove this line now

Reply:
Are you talking about removing this whole thing:

```
// @param _padding: Nested public functions always get called with MAX_ARGS, since we don't have the ABI available
    // during execution time to know how many args are expected, hence the _padding argument. We should
    // be able to remove it when we migrate to brillig.
```

or just this sentence:
`We should  be able to remove it when we migrate to brillig.`

My knowledge is not up to date on this.